testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistProductBomJars.java

```diff
@@ -13,6 +13,8 @@ public enum WhitelistProductBomJars {
             "com.nimbusds.oauth2-oidc-sdk",
             "com.nimbusds.content-type",
             "io.github.crac",
+            "net.java.dev.jna.jna",
```

Comment:
Can you add a comment about where these deps come from?

Reply:
Updated